### PR TITLE
Adding load balancer doc

### LINKED ISSUE #477: Explain the configuration of a load balancer in a cluster environment
Hi team,

I just realized that the [cluster section](https://documentation.wazuh.com/current/user-manual/manager/wazuh-cluster.html) of the documentation doesn't currently have any explanation of the configuration of a load balancer in a Wazuh cluster environment. In my honest opinion, it would be necessary to offer a guide for this in order to clarify how the cluster works.

Regards

### Pending tasks

- [x] Research about load balancing. Decide about what's the best option to explain this concept on the documentation.
- [x] Try to configure a load balancer for a Wazuh cluster.
- [x] Elaborate a list of required steps to configure this. The list must be minimal but sufficient to properly configure the feature.
- [x] Structure and document the steps in a proper, user-readable format.
- [x] Revision: grammar, readability, format, document structure.

### PR DESCRIPTION
**Pull request by JaoChaos**
Closes #477 

Added how to configure the load balancer.

### CODE REVIEW

File: source/user-manual/manager/wazuh-cluster.rst [hunk 1]
```diff
@@ -189,7 +189,42 @@ The following must be considered when configuring a Wazuh app:
...
+
+The correct way to use it is to point every agent to send the alerts to the *load balancer*:
+
+1. Edit the Wazuh agent configuration in ``/var/ossec/etc/ossec.conf`` to add the **Load Balancer** IP address. In the ``<client><server>`` section, change the ``LOAD_BALANCER_IP`` value to the ``load balancer`` address and ``port``:
+
+  .. code-block:: xml
+
+    <client>
+      <server>
+        <address>LOAD_BALANCER_IP:PORT</address>
```
Comment (jesusgn90) [inline source/user-manual/manager/wazuh-cluster.rst]: We don't set the port here, just the IP

File: source/user-manual/manager/wazuh-cluster.rst [hunk 2]
```diff
@@ -189,7 +189,42 @@ The following must be considered when configuring a Wazuh app:
 Pointing agents to the cluster with a load balancer
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The best setup to report agents' events to a cluster is a load balancer. Keep in mind the following considerations:
+A **load balancer** is a service that distributes de workloads across multiple resources.
+In Wazuh's case, users want to use a load balancer to catch all the agent's alerts and distribute them between the different workers in the cluster.
+
+The correct way to use it is to point every agent to send the alerts to the *load balancer*:
```
Comment (jesusgn90) [inline source/user-manual/manager/wazuh-cluster.rst]: "alerts" -> "events"

**Comment by JaoChaos** [comment]
Fixed grammar.

Tested load balancer in local.

**Comment by JaoChaos** [comment]
Tested Load Balancer in local with different Vagrant Machines and it worked perfectly.

**Pull request merged**